devices/Bmm150/Bmm150TrimRegister.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Magnetometer
+{
+    /// <summary>
+    ///  Represents the trim registers of the sensor (trim values in the "trim_data" of device structure).
+    /// </summary>
+    public class Bmm150TrimRegisterData
+    {
+        /// <summary>
+        /// trim dig_x1 data
+        /// </summary>
+        public byte dig_x1 { get; set; }
```

Comment:
please follow the naming convention for all those as well. so `DigX1` in this case. Same for the others.

Reply:
done